deploy/osd-25821-capa-annotator/should_patch.py

```diff
@@ -0,0 +1,43 @@
+import sys
```

Comment:
I assume we are duplicating the shell-script and python-script, so it's:
1. Easier to review.
2. There are tests for the Python part.
If so - and the actual code that matters - is just the one in the ConfigMap, I'd like to see a comment explaining this.

Reply:
I added a script for generating the ConfigMap, as well as added some more comments and clarified in the README.